model/project_ref.go

```diff
@@ -1266,6 +1277,17 @@ func FindProjectRefsQ(filter bson.M) ([]ProjectRef, error) {
 	return addLoggerAndRepoSettingsToProjects(projectRefs)
 }
 
+func FindProjectRefsQ(filter bson.M) ([]ProjectRef, error) {
```

Comment:
Could  this could take in a boolean `merged` to determine whether to call addLoggerANdRepoSettings as opposed to having both functions do similar things? We could also make this private to make it more obvious this is just a helper function.